charts/ccsm-helm/README.md

```diff
@@ -15,84 +32,232 @@ This functionality is in beta and is subject to change. The design and code is l
 
 ## Installing
 
-* Add the official Camunda Cloud helm charts repo
-
 ```shell
+  # Add the official Camunda Cloud helm charts repo
```

Comment:
:wrench: I would put the explanations above in simple text, explaining what the snippet below does, and leave no comments in the snippet. The main reason for this is so I can then copy paste it to my console (which I know is very bad and evil of course :smile:)